conjur-api/ResourceMetadata.cs

```diff
@@ -0,0 +1,65 @@
+﻿// <copyright file="ResourceMetadata.cs" company="Conjur Inc.">
+//     Copyright (c) 2016 Conjur Inc. All rights reserved.
+// </copyright>
+// <summary>
+//     Class representing resource metadata returned from web request.
+// </summary>
+
+namespace Conjur
+{
+    using System.Runtime.Serialization;
+
+    [DataContract]
+    public class ResourceMetadataֿ
```

Comment:
This class name has a funny character at the end. It might make it hard to type.

Reply:
Great catch! It made me crazy. Still getting use to write in Mac :)